DigitalLearningSolutions.Web/Helpers/CustomPolicies.cs

```diff
@@ -5,7 +5,7 @@
     public class CustomPolicies
     {
         public const string UserOnly = "UserOnly";
-        public const string UserCentreAdminOnly = "UserCentreAdminOnly";
+        public const string UserCentreAdmin = "UserCentreAdminOnly";

```

Comment:
Do we also want to change the string value?

Reply:
I decided not to because I didn't know if it interacted with anything besides this, but after checking I don't think it does.